tests/unit/test_parameters/test_process_parameter_data.py

```diff
@@ -2,56 +2,59 @@
 # Tests for the parameter processing functions
 #
 
-
 import os
 import numpy as np
 import pybamm
-
 import pytest
 
 
+@pytest.fixture
+def path():
+    return os.path.abspath(os.path.dirname(__file__))
```

Comment:
Could we rename this to `parameters_path` so that it is slightly more clear what this refers to?

```suggestion
def parameters_path():
    return os.path.abspath(os.path.dirname(__file__))
```